src/plugins/intel_gna/src/avx2.cpp

```diff
@@ -9,6 +9,14 @@
 
 #    include "preprocessing.hpp"
 
+inline float getSingleValueFromM256Vector(const __m256& vector, const uint32_t index) {
+#if defined(_MSC_VER)
+    return vector.m256_f32[index];
+#else if defined(__GNUC__)
```

Comment:
does it cover all AVX2 cases? I mean, in cmake ie_avx2_optimization_flags macro I can also see "Intel"

Reply:
You're right. I've pushed it only as a test to see if it will compile on Linux. Turns out, I need a different solution anyway